test/e2e/registrymirror.go

```diff
@@ -13,8 +13,8 @@ func runRegistryMirrorConfigFlow(test *framework.ClusterE2ETest) {
 	test.ExtractDownloadedArtifacts()
 	test.DownloadImages()
 	test.ImportImages()
-	test.CreateCluster()
-	test.DeleteCluster()
+	test.CreateCluster(framework.WithBundlesOverride("./eks-anywhere-downloads/bundle-release.yaml"))
```

Comment:
Do we download bundle release file for each test so `./eks-anywhere-downloads/bundle-release.yaml` this path will always have the bundle release file?

Reply:
yeah the bundle release file gets saved in that path when ExtractDownloadArtifacts() runs